pkg/cmd/roachtest/cluster.go

```diff
@@ -1387,7 +1387,7 @@ func (c *clusterImpl) FetchDebugZip(
 	nodes := selectedNodesOrDefault(opts, c.All())
 
 	// Don't hang forever if we can't fetch the debug zip.
-	return timeutil.RunWithTimeout(ctx, "debug zip", 5*time.Minute, func(ctx context.Context) error {
+	return timeutil.RunWithTimeout(ctx, "debug zip", 10*time.Minute, func(ctx context.Context) error {
```

Comment:
Seems reasonable, considering how long debug zip generation can take on larger clusters. The side-effect is we may end up waiting 10 minutes in case a node is truly unreachable.